### PR TITLE
feat cart page

### PR DESCRIPTION
**Pull request by khyatibaria**
### Created Cart Page
Please review
working link:-https://deploy-preview-4--bookish-club.netlify.app/cart

### CODE REVIEW

File: assets/profile.svg [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (poojavishnoi) [inline assets/profile.svg:43]: Instead of using svg files for icon, you can use font awesome for icons

File: cart.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+.item-lists {
+  display: inline-block;
+  padding: 0.5rem;
+  border-bottom: 1px solid var(--grey);
+}
+
+.order-btn {
+  width: 100%;
+  margin: 0px;
```
Comment (poojavishnoi) [inline cart.css:58]: you can just write 0 here.

**Review by poojavishnoi — APPROVED** [review]
Looks good.Added few suggestion.

**Review by NeerajVodala — APPROVED** [review]
Everything looks good.
All the best.

File: cart.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  font-weight: 600;
+  background-color: var(--primary-color);
+}
+
+.price {
+  float: right;
```
Comment (LordUttam) [inline cart.css:64]: Suggestion: Avoid using floats. Make item-lists a flex container and use space-between.

File: cart.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+</head>
+
+<body>
+
+    <div class="grid-container">
+        <div class="navigation grid-item">
+            <nav class="navigation">
+                <div class="navigation-bar">
+                    <div class="navigation-logo">
+                        <!-- <img src="./logo.svg" /> -->
```
Comment (LordUttam) [inline cart.html:21]: remove comments

**Review by LordUttam — COMMENTED** [review]
Looks good to me

**Pull request merged**